students/ChelseaSmith/lesson8/circle.py

```diff
@@ -0,0 +1,88 @@
+# lesson 8 assignment to make a circle class
+import math
+
+class Circle:
+
+    def __init__(self, radius):
+        self.radius=radius
+
+    @property
+    def diameter(self):
+        return 2*self.radius
+    @diameter.setter
+    def diameter(self, val):
+        self.radius = val / 2
+
+    @property
+    def area(self):
+        return math.pi * self.radius ** 2
+
+
```

Comment:
your class methods should have one line space between them (where module functions have 2 lines).